locations/gruni73.yml

```diff
@@ -123,8 +130,58 @@ networks:
     mesh_iface: mesh
     mesh_metric_lqm:
       - default 0.4 # penalty on wild meshing
+  - vid: 50
+    name: prdhcp
+    role: uplink
```

Comment:
Need `role: dhcp` here and the convention is vlan 41 - uplink and 50 would be for tunspace

Reply:
dhcp does not work as rule, since it expects a prefix and so on. I just want to have a untagged switch bridge.

Reply:
Ah - then use `role: ext` with untagged :)

Reply:
ah thanks! and vlan?

Reply:
thanks. i would prefer untagged: true since i just wan all ports to be untagged, regardeless what router is used and to what router we will siwtch in the future.
I think the comment is wrong in this examples. ;) no isolation und so is happening there.